src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -158,7 +168,6 @@
                       "property": "addressLine1",
                       "dataType": { "dataType": "stringType" },
                       "queryable": true,
-                      "valueGetter": "( SELECT array_agg(elems.value ->> 'addressLine1') FROM jsonb_array_elements(org.jsonb->'addresses') AS elems)",
```

Comment:
We don't retrieve values of these inner columns (only the outer one, which gives back a JSON blob), so these serve zero purpose.